.circleci/config.yml

```
@@ -194,6 +194,19 @@ jobs:
           name: check for inconsistency between schema ddl & graphql
           command: git diff --exit-code || (echo 'The contents of the `schema` folder are out of sync with `app/server/schema.graphql`. Did you forget to \"yarn build:relay\"?' && false)
 
+  compile:
+    machine:
+      image: ubuntu-1604:201903-01
+    working_directory: ~/cas-ciip-portal/app
+    steps:
+      - attach_workspace:
+          at: ~/
+      - run:
+          name: Audit Dependencies
+          command: |
+            source ~/.asdf/asdf.sh
+            yarn build # ensure next.js can compile a production build
```

```suggestion
            yarn build:relay
            yarn build:next # ensure next.js can compile a production build
```
This is what we're doing on pathfinder. See the s2i assemble script

---

good point, the schema is checked in a separate, isolated step